pkg/app/rds_postgres.go

```diff
@@ -131,6 +130,17 @@ func (pdb *RDSPostgresDB) Install(ctx context.Context, ns string) error {
 		return err
 	}
 
+	pdb.testWorkloadName = fmt.Sprintf("%s-workload", pdb.name)
+
+	testDeployment := bastionWorkload(ctx, pdb.testWorkloadName, "postgres", pdb.namespace)
+	_, err = pdb.cli.AppsV1().Deployments(pdb.namespace).Create(context.Background(), testDeployment, metav1.CreateOptions{})
```

Comment:
```suggestion
	_, err = pdb.cli.AppsV1().Deployments(pdb.namespace).Create(ctx, testDeployment, metav1.CreateOptions{})
```
please pass parent context instead of initializing new one